output/intrinsic-perk-lookup-V2.json

```diff
@@ -0,0 +1,97 @@
+{
+  "5": [
+    [878286503, 3755070117],
+    [1294026524, 944506345, 1484442054, 3610814281],
+    [1458010786, 1636108362],
+    [1019291327, 3208839961]
+  ],
+  "6": [
+    [-99999999, 1791592647, 2770223582],
+    [-7777777, 1458010786, 2189829540, 1322370662, 4045839491, 2144092201, 1863355414],
+    [1294026524, 647617635],
+    [3468089894, 3923638944, 507151084, 1030990989],
+    [-99999999, 3174300811]
+  ],
+  "7": [
+    [878286503],
+    [1458010786, 4208418110, 4004944400],
+    [1294026524, 3905543891],
+    [1019291327, 2307143135],
+    [2874284214]
+  ],
+  "8": [
+    [3920852688, 3364911712, 4185339856],
+    [1458010786, 377257911, 2741975068],
+    [1636108362],
+    [1019291327, 1000724343, 3668782036]
+  ],
+  "9": [
+    [878286503],
+    [1294026524],
+    [-99999999, 656200654],
+    [1636108362, 1186480754],
+    [-99999999, 1927916065],
+    [1019291327, 1656957541],
+    [-99999999, 1657056865]
+  ],
+  "10": [
+    [878286503, 938999636],
+    [1294026524, 1070100196],
+    [3468089894, 3081173348],
+    [-99999999, 281315705]
+  ],
+  "11": [
+    [996573084, 1458010786, 1210807262],
+    [-99999999, 481338655],
+    [895140517, 918679156, 1636108362],
+    [-99999999, 1394384862],
+    [3468089894, 3983457027],
+    [-99999999, 372430833, 3751354116]
+  ],
+  "12": [[878286503], [1294026524, 4148158229], [1019291327], [-99999999, 2261491232]],
+  "13": [
+    [-7777777, 3419274965, 216781713, 1019291327, 2491817779, 411799453],
+    [1294026524],
+    [3468089894],
+    [-99999999, 3649430342],
+    [-99999999, 2473404935]
+  ],
+  "14": [
+    [806997698],
+    [1458010786],
+    [1282254042, 2121086290, 334466122],
+    [1636108362],
+    [3330548924],
+    [-7777777, 1294026524, 3449390870, 31057037]
+  ],
+  "54": [
+    [1713206376],
+    [-7777777, 4172267910],
+    [-7777777, 4172267910, 3642491337, 4072668471],
```

Comment:
`[-7777777, 4172267910]` should be removed as it is contained in the next line completely